### PR TITLE
Merge form

### PR DESCRIPTION
**Pull request by acflinch**

### CODE REVIEW

File: code/frontend/public/submitform.html [hunk 1]
```diff
@@ -1,73 +1,182 @@
-<!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>External Form</title>
+    <title>Search Page</title>
 </head>
+
+<style>
```
Comment (jshilts657) [inline code/frontend/public/submitform.html]: should move all styles to style.css file

File: code/frontend/public/submitform.html [hunk 2]
```diff
@@ -1,73 +1,182 @@
...
               <option value="test">test data</option>
             </select>
         </div>
-        <button type="submit">Generate</button>
+        <br>
+        <div class="buttons">
+
+            <div class="action_btn">
+
+                <button name="submit" class="action_btn submit" type="submit" value="Submit" onclick="myFunction()">Submit</button>
```
Comment (jshilts657) [inline code/frontend/public/submitform.html:93]: try to refactor some of these styles to css

File: code/frontend/node_modules/.cache/default-development/index.pack
Comment (jshilts657) [inline code/frontend/node_modules/.cache/default-development/index.pack:1]: please remove this file and add it to the .gitignore file
This is part of gits internal storage mechanism.

File: code/frontend/src/tests/submitform.test.js
Comment (jshilts657) [inline code/frontend/src/tests/submitform.test.js:1]: Test looks good but I think we are also at a point where we  need to update the SDD based on the unit test work the team has done. Specifically in regards to automation

**Review by jshilts657 — CHANGES_REQUESTED** [review]
Got a few things to clean up and document otherwise looks good

File: code/frontend/src/tests/submitform.test.js
Comment (acflinch) [reply code/frontend/src/tests/submitform.test.js:1]: i'll try to find some time tomorrow to add the unit test done so far to the SDD

File: code/frontend/public/submitform.html (hunk 2 above)
Comment (acflinch) [reply code/frontend/public/submitform.html:93]: i'm not sure what all in here is style, James worked on that portion. the only thing i think that attribution to the style are those class tags which are linking to style under those class names in the css

Comment (jshilts657) [reply code/frontend/public/submitform.html:93]: You are correct, apologies

File: code/frontend/public/styles.css
Comment (jshilts657) [inline code/frontend/public/styles.css:1]: Css files need to be in the asset folder 
code/frontend/src/assets

**Review by jshilts657 — CHANGES_REQUESTED** [review]
Everything looks now just need to move the css file to the correct location and update your imports.

Just PR comment for more details

**Review by jshilts657 — APPROVED** [review]
Looks good

**Review by BlueGreenDragon — APPROVED** [review]
Looks good.

**Review by rzheng2019 — APPROVED** [review]

**Pull request merged**